### ISSUE TITLE
Write a real README

### ISSUE THREAD

**Issue by tmatsuo** [GoogleCloudPlatform/appengine-php#3]
What is this repo?
Why do you need this?
How to compile? Which configuration options are needed?
Do I need to compile libmcrypto?
Do I need to compile openssl?

**Comment by jmtorrescusa** [comment]
I totally agree. This excerpt from [google documentation](https://cloud.google.com/appengine/downloads?hl=es#Google_App_Engine_SDK_for_PHP) gives no clue about how to install appengine-php:

> Build and install the [PHP interpreter](https://github.com/GoogleCloudPlatform/appengine-php) and [App Engine PHP extension](https://github.com/GoogleCloudPlatform/appengine-php-extension). Specify the path to `php-cgi` and `gae_runtime_module.so` when [running the development server](https://cloud.google.com/appengine/docs/php/tools/devserver?hl=es#PHP_Running_a_custom_php_install).

They do explain how to install mysql-server-5.5 with apt-get (so trivial), but don't explain this confusing point.

**Comment by v0lsh3bnik** [comment]
There is still no real README. Have any of you found how to use this?
